providers/curation/github.js

```diff
@@ -256,7 +257,14 @@ ${this._formatDefinitions(patch.patches)}`
   }
 
   _formatDefinitions(definitions) {
-    return definitions.map(def => `- ${def.coordinates.name} ${Object.keys(def.revisions)[0]}`)
+    return definitions.map(
+      def =>
+        `- [${def.coordinates.name} ${
+          Object.keys(def.revisions)[0]
+        }](https://clearlydefined.io/definitions/${EntityCoordinates.fromObject(def.coordinates)}/${
```

Comment:
@storrisi, it works for production environment. Is it possible to make it work also for dev environment (dev.clearlydefined.io)?

Reply:
Reopened https://github.com/clearlydefined/service/issues/540. Merging to prod for now.